packages/components/src/components/Trigger/Trigger.test.js

```diff
@@ -90,7 +90,13 @@ const fakeTrigger = {
     },
     {
       cel: {
-        filter: 'cel-filter'
+        filter: 'cel-filter',
+        overlays: [
```

Comment:
the tests should verify this new content is handled correctly, same for the others